dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/ACVoltageControl/QRefQU.mo

```diff
@@ -48,12 +48,13 @@ model QRefQU "Function that calculates QRef for the Q mode and the U mode depend
     Placement(visible = true, transformation(origin = {-85, 20}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Modelica.Blocks.Nonlinear.SlewRateLimiter slewRateLimiter1(Rising = SlopeQRefPu)  annotation(
     Placement(visible = true, transformation(origin = {-70, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Blocks.Continuous.PIAntiWindup PI(Ki = KiACVoltageControl, Kp = KpACVoltageControl, uMax = QMaxCombPu, uMin = QMinCombPu, integrator.y_start = Q0Pu)  annotation(
-    Placement(visible = true, transformation(origin = {25, 20}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
 
   parameter Types.ReactivePowerPu Q0Pu "Start value of reactive power in pu (base SNom) (generator convention)";
   parameter Types.VoltageModulePu U0Pu "Start value of voltage amplitude in pu (base UNom)";
-
+  Blocks.Continuous.PIAntiWindupInput pIAntiWindupInput(Ki = KiACVoltageControl, Kp = KpACVoltageControl, uMax = 0.4, uMin = -0.4)  annotation(
```

Comment:
Why 0.4 and -0.4 ? Also, y_start missing

Reply:
Moreover, please remove one blank space before annotation

Reply:
I have added Y0 = Q0Pu. and replaced 0.4 and -0.4 by QMaxCombPu and QMinCombPu respectively. Is that a correct choice ?